.github/workflows/release.yml

```diff
@@ -43,9 +44,9 @@ on:
       - '**[0-9]+.[0-9]+.[0-9]+*'
 
 jobs:
-  # Run 'cargo dist plan' (or host) to determine what tasks we need to do
+  # Run 'dist plan' (or host) to determine what tasks we need to do
   plan:
-    runs-on: ubuntu-latest
+    runs-on: "ubuntu-20.04"
```

Comment:
Do we really need to target a specific version of Ubuntu, especially such an old one? (two LTS releases ago!)

Reply:
This was generated by `cargo dist`. I checked their issue [on updating the runner to the LTS latest](https://github.com/axodotdev/cargo-dist/issues/1613), but their reasoning was to further improve compatibility. I don't think that we'd benefit from it. I'd advise to go away from `cargo dist` and build our own CI with `cargo cross` (a cross platform compiler)